style_rwd.css

```diff
@@ -2,21 +2,233 @@
     img.main_photo {
         display: none;
     }
-    .catch_phrase{
-        max-width: 767px;
+    h2 {
+        margin-block-start: 0;
+        margin-block-end: 0;
+    }
+    .nav_margin {
+        margin: 0 ;
+    }
+    .margin_header{
+        margin: 0;
+    }
+    .margin_menu {
+        margin: 0;
+    }
+    .header_wrapper>div {
+        padding: 5px 0 5px 10px;
+    }
+    .header_wrapper{
+        max-width:767px;
+        max-height:50px;
+    }
+    .section_wrapper{
+        max-width:767px;
+        margin:0 auto;
+        padding: 0;
+    }
+    .font_size{
+        font-size:10px;
+    }
+    .section_hero {
+        padding-bottom: 42px;
     }
     .column{
+        max-width:767px;
         align-items: center;
         justify-content: center;
     }
-    .invite_phrase{
+    .catch_phrase .invite_phrase{
         text-align: center;
-        font-size: 5px;
-        font-weight: 30;
-        max-width: 767px;
+        font-size: 40px;
+        margin-bottom: 0;
+        max-width:767px;
+    }
+    .button_container {
+        justify-content: flex-start;
+        max-width:767px;
+        margin: 0 0 0 100px;
```

Comment:
why this margin is set to 100px bottom?

Reply:
this one is set to 100 px left. If i try to change this on margin-left: 100px; this one is jumping down for some reason

Reply:
Find out what that reason is and add a fix commit :)

Reply:
fixed now I guess